UKHomeOffice/sgmr-service
# SPCR-211 Fix year validation for values far into the future

## Description
If a user typed in a date with a year extremely far into the future (e.g. 7000) it was validated by the service. This should not have happened, and additional validation has been implemented to ensure the year values input are within a reasonable range.

## To Test
- Go to the People Page
- Add a new person
- Fill out all the relevant details except the document expiry year
- Add a document expiry year above 2099
- You should get an error
- Add a document expiry year between 1900 and 2099
- The new person should be created without a problem

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
